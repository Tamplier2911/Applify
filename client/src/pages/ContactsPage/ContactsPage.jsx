// import "./ContactsPage.scss";
import React from "react";

// components
import Contacts from "../../components/Contacts/Contacts";

// JS Rendering CSS
import {
  ContactsPageContainer,
  ContactsPagePlaceholder,
} from "./ContactsPageStyles";

const ContactsPage = () => {
  return (
    <ContactsPageContainer data-test="contacts-page">
      <Contacts data-test="contacts-page-cp" />
      <ContactsPagePlaceholder />
    </ContactsPageContainer>
  );
};

export default ContactsPage;
