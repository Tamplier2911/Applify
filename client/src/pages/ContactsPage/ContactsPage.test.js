import React from "react";
import toJson from "enzyme-to-json";

// component
import ContactsPage from "./ContactsPage";

// utils
import { setShallow, findByTestAttr } from "../../tests/testUtils";

describe("<ContactsPage />", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = setShallow(<ContactsPage />);
  });

  it("renders without error", () => {
    const contactsPage = findByTestAttr(wrapper, "contacts-page");
    expect(contactsPage.length).toEqual(1);
  });

  it("renders children", () => {
    const contacts = findByTestAttr(wrapper, "contacts-page-cp");
    expect(contacts.length).toEqual(1);
  });

  it("matches snapshot", () => {
    const contactsPage = findByTestAttr(wrapper, "contacts-page");
    expect(toJson(contactsPage)).toMatchSnapshot();
  });
});
