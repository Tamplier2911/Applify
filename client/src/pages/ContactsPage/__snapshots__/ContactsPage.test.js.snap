// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ContactsPage /> matches snapshot 1`] = `
<styled.section
  data-test="contacts-page"
>
  <Connect(Contacts)
    data-test="contacts-page-cp"
  />
  <styled.div />
</styled.section>
`;
